### PR TITLE
G#1: Initial backend commit

### PR DESCRIPTION
**Pull request by SidhaantAnand**
Issue: user/sid/G#1

Created an initial backend service structure

### CODE REVIEW

File: back/src/app.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+app.use(bodyParser.urlencoded({
+  extended: true
+}));
+app.use(bodyParser.json())
+
+app.use(session({secret: 'secret'}));
+
+const server = app.listen(process.env.PORT || 8080, () => { 
+	if(!process.env.SPOTIFY_CLIENT_ID || !process.env.SPOTIFY_CLIENT_SECRET) {;
```
Comment (sohinim98) [inline back/src/app.js:15]: remove semi-colon.

Comment (SidhaantAnand) [reply back/src/app.js:15]: okay nice catch

File: back/src/app.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+		server.close();
+		return;
+	}
+	console.log(`Listening on port ${process.env.PORT || 8080}!`);
```
Comment (sohinim98) [inline back/src/app.js:19]: Don't keep this in prod.

Comment (sohinim98) [reply back/src/app.js:19]: Or at least remove the exclamation mark for now.

**Pull request merged**